BondageClub/Scripts/Dialog.js

```diff
@@ -117,32 +117,32 @@ var DialogSelfMenuOptions = [
 /**
  * Compares the player's reputation with a given value
  * @param {string} RepType - The name of the reputation to check
- * @param {string} Value - The value to compare
+ * @param {number} Value - The value to compare
  * @returns {boolean} - Returns TRUE if a specific reputation type is less or equal than a given value
  */
 function DialogReputationLess(RepType, Value) { return (ReputationGet(RepType) <= Value); }
 
 /**
  * Compares the player's reputation with a given value
  * @param {string} RepType - The name of the reputation to check
- * @param {string} Value - The value to compare
+ * @param {number|string} Value - The value to compare
  * @returns {boolean} - Returns TRUE if a specific reputation type is greater or equal than a given value
  */
 function DialogReputationGreater(RepType, Value) { return (ReputationGet(RepType) >= Value); }
 
 /**
  * Compares the player's money with a given amount
- * @param {string} Amount - The amount of money that must be met
+ * @param {number|string} Amount - The amount of money that must be met
```

Comment:
`CommonDynamicFunctionParams` (see above)
```suggestion
 * @param {string} Amount - The amount of money that must be met
```